unity3d/Assets/Plugins/iOS/swrve_sdk.mm

```diff
@@ -1,5 +1,8 @@
 #import <CoreTelephony/CTTelephonyNetworkInfo.h>
 #import <CoreTelephony/CTCarrier.h>
+#if !defined(SWRVE_NO_IDFA)
```

Comment:
Not really anything, but why `#if !defined` and not `#ifndef`?

When does `SWRVE_NO_IDFA` ever get defined?